### PR TITLE
Ported SpecFlowHelper to FAKE 5

### PR DESCRIPTION
**Pull request by magicmonty**
Ported SpecFlowHelper to FAKE 5 as Fake.DotNet.Testing.SpecFlow

### CODE REVIEW

**Comment by magicmonty** [comment]
@matthid Added Documentation page

File: src/app/Fake.DotNet.Testing.SpecFlow/SpecFlow.fs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+open Fake.Core
+open Fake.IO
+open Fake.IO.Globbing
+open Fake.IO.FileSystemOperators
+open System.IO
+open System.Text
+
+/// SpecFlow execution parameter type.
+[<CLIMutable>]
```
Comment (matthid) [inline src/app/Fake.DotNet.Testing.SpecFlow/SpecFlow.fs]: We usually no longer specify CLIMutable in Fake 5

File: src/app/Fake.DotNet.Testing.SpecFlow/SpecFlow.fs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+// Runs SpecFlow on a project.
+/// ## Parameters
+///
+///  - `setParams` - Function used to manipulate the default SpecFlow parameter value.
+let generate setParams =    
+    let parameters = setParams SpecFlowDefaults
+
+    Trace.traceStartTaskUnsafe "SpecFlow " parameters.SubCommand
```
Comment (matthid) [inline src/app/Fake.DotNet.Testing.SpecFlow/SpecFlow.fs]: We can use `use _ = Trace.traceStart ...` here to no longer be 'unsafe' :)

Comment (magicmonty) [reply src/app/Fake.DotNet.Testing.SpecFlow/SpecFlow.fs]: Ok, thanks for the hint. Will fix that on the other PRs too

**Comment by matthid** [comment]
Thanks!

**Pull request merged**